core/src/scene/scene.cpp

```diff
@@ -83,6 +83,49 @@ const Style* Scene::findStyle(const std::string& _name) const {
 
 }
 
+std::vector<const SceneLayer*> Scene::getLayerHierarchy(const std::string& _name) const {
+
+    std::vector<const SceneLayer*> layers;
+
+    const std::string delimiter = ":";
```

Comment:
Defining a delimiter in several places like this is fragile, I'd suggest defining the delimiter once in a public variable associated with a relevant class.